data/api/server-server/public_rooms.yaml

```diff
@@ -147,69 +148,11 @@ paths:
         required: true
       responses:
         "200":
-          description: A list of the rooms on the server.
+          description: A filtered list of the published rooms on the server.
           content:
             application/json:
               schema:
-                type: object
-                description: A list of the rooms on the server.
-                required:
-                  - chunk
```

Comment:
AFAICT this is the only difference to `public_rooms_response.yaml`. I cannot imagine that `chunk` was supposed to be required on only one of the four endpoints so I have made this reuse the shared schema.